### PR TITLE
discord-botlist.eu

### PR DESCRIPTION
**Pull request by MauricePascal**

### CODE REVIEW

File: websites/D/discord-botlist/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "MauricePascal#3009",
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:4]: Remove discriminator, aka the `#3009`

File: websites/D/discord-botlist/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your free Botlist"
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:9]: Use this instead: `Your Discord Bot List! We give you the avaibillity to apply your bot and give your bot a page on our website.`

File: websites/D/discord-botlist/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "service": "discord-botlist",
+    "version": "1.0",
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:12]: Make this `1.0.0`

File: websites/D/discord-botlist/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "logo": "https://discord-botlist.eu/pics/logo.jpg",
+    "thumbnail": "https://discord-botlist.eu/pics/logo.jpg",
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:14]: Upload these to imgur

File: websites/D/discord-botlist/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "color": "#001FFF",
+    "tags": [
+      "discord",
+      "discord bot list",
+      "discord bot",
+      "bot"
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:20]: Remove the spaces, make them dashes

File: websites/D/discord-botlist/presence.ts [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+    if (presenceData.details == null) {
+        //This will fire if you do not set presence details
+        presence.setTrayTitle(); //Clears the tray title for mac users
+        presence.setActivity(); /*Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name*/
+    } else {
+        //This will fire if you set presence details
+        presence.setActivity(presenceData); //Update the presence with all the values from the presenceData object
+    }
+});
```
Comment (Alanexei) [inline websites/D/discord-botlist/presence.ts:84]: Remove all of the comments

**Comment by MauricePascal** [comment]
Capture
Capture2
Capture3

File: websites/D/discord-botlist/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your Discord Bot List! We give you the avaibillity to apply your bot and give your bot a page on our website."
+    },
+    "service": "discord-botlist",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/GqUUpuX",
+    "thumbnail": "https://i.imgur.com/GqUUpuX",
+    "color": "#001FFF",
+    "tags": ["discord", "discord bot list", "discord bot", "bot"],
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:16]: Remove the spaces, replace them with dashes.

File: websites/D/discord-botlist/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "MauricePascal",
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your Discord Bot List! We give you the avaibillity to apply your bot and give your bot a page on our website."
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:9]: I'm noticing this now: change `avaibillity` to `ability`

File: websites/D/discord-botlist/dist/metadata.json [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your Discord Bot List! We give you the ability to apply your bot and give your bot a page on our website."
+    },
+    "service": "discord-botlist",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/GqUUpuX",
+    "thumbnail": "https://i.imgur.com/GqUUpuX",
```
Comment (Alanexei) [inline websites/D/discord-botlist/dist/metadata.json:14]: Make the thumbnail a screenshot of the website,.

Comment (Alanexei) [reply websites/D/discord-botlist/dist/metadata.json:14]: image
also fix this ^^

**Comment by MauricePascal** [comment]
New Design
Capture

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by Bas950** [comment]
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

File: websites/D/discord-botlist/dist/metadata.json [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your Discord Bot List! We give you the ability to apply your bot and give your bot a page on our website."
+    },
+    "service": "discord-botlist",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/GqUUpuX",
+    "thumbnail": "https://i.imgur.com/xVGm3FN",
```
Comment (frozaum) [inline websites/D/discord-botlist/dist/metadata.json:14]: Please use the direct links of the Imgur links.

Comment (frozaum) [reply websites/D/discord-botlist/dist/metadata.json:14]: also update your fork.

Comment (MauricePascal) [reply websites/D/discord-botlist/dist/metadata.json:14]: But alan said I should do it this way

Comment (MauricePascal) [reply websites/D/discord-botlist/dist/metadata.json:14]: But okay

**Comment by doomlerd** [comment]
Update your fork again, it's 71 commits behind

**Pull request closed without merging**